src/lang.js

```diff
@@ -8,6 +8,42 @@
    * @words expect array
    *
    */
+  
+
+  let setLangType = function(LangType) {
+    let currentLangType;
+    //  Check language type is present in congreaLanguages or not.
+    if(window.congreaLanguages.hasOwnProperty(LangType)) {
+      currentLangType = LangType;
+    } else if(LangType.length>=4) {
+      // check language by first two characters of langtype
+      LangType = LangType.substring(0,2);
+      if(window.congreaLanguages.hasOwnProperty(LangType)) {
+        currentLangType = LangType;
+      } else {
+        currentLangType = "en";
+      }
+    } else {
+      // used default language
+      currentLangType = "en";
+    }
+    return currentLangType;
+  }
+
+  const getLang = function (Langtype){
+     if(typeof Langtype === "undefined" || Langtype === "none") { 
+      // language is not passed from the moodle
+      let userBrowserLang = window.navigator.language;
+      Langtype = userBrowserLang.toLowerCase();
+      Langtype = Langtype.replace(/-/g, '_' ); // Replacing hyphen with underscore
+     }
+     const congreaLang = setLangType(Langtype);
+     window.message = window.congreaLanguages[congreaLang];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment to property of function parameter 'window'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)